### DISCUSSION TITLE
About recommended hardware for Hubs

### DISCUSSION THREAD

**Discussion by jasoncks** [mozilla/hubs#3290] [Hubs Cloud Questions] [answered]
In this [page](https://hubs.mozilla.com/docs/hubs-create-join-rooms.html), it mentions about the software side, is there any recommendation in hardware specification in using Hubs, such as CPU, Ram and even Graphic board?
I know it is depends on the scene contents, but do Hubs has any minimum target requirement for hardware? (PC, mobile and VR Device)

**Comment by emclaren** [comment]
Hi Jason, like in https://github.com/mozilla/hubs/discussions/3289 this is a good question, but very challenging to answer. There are no firm minimum requirements in terms of hardware specs. Performance varies widely based on not only the composition of the scene, the other avatars in the space, and the type of media shared, but also depending on the browser used and other software installed on the system. To maximize the number of devices that a room loads properly on, we recommend optimizing your 3D assets & environment for the web, and keeping room media (images/music/videos) to a minimum.

**Reply by Alexjkz** [reply]
Well, but considering the maximum capacity of 25 avatars, and a pre-set scene, such as the meeting hall (but without additonal object), what are the system requirements that allow to run this flawlessly?

**Comment by Alexjkz** [comment]
Also, the performance are strange, i had 100% CPU usage, with bad performances on a 7th gen desktop i5 (the same desktop was also equipped with a dedicated GPU GTX 980ti that wasn't used), while, in the same scene another user was on mobile (dont'know the specific smartphone model) and his performances where good

**Comment by pattersonbl2** [comment, marked as answer]
Currently, We don't have a min because most computers now are able to handle the web based deployment. performance  can be a factor or resources in AWS or networking as well.  I would say i5 and 16gb of ram should be enough to run it.  I am currently running a local deployment on a m1 Mac
